interfaces/input_parameters/__ring_options.py

```diff
@@ -35,6 +35,11 @@
 from ...utilities import timing
 
 
```

Comment:
Worth adding a deprecation warning that gets thrown at import?

Reply:
We can, I'd go for a more drastic approach though to list all the functionalities of RingOptions that would be missing in the new implementation and then delete the module since it cannot really work on its own.

Reply:
Fine with me